test/alternator/run

```diff
@@ -115,6 +115,19 @@ run.wait_for_services(pid, [
     lambda: check_alternator(alternator_url),
 ])
 
+# Set up the the proper authentication credentials needed by the Alternator
+# test. Currently this can only be done through CQL, which is why above we
+# needed to make sure CQL is available.
+cluster = run.get_cql_cluster(ip)
+cql = cluster.connect()
+
+# Additional role and service level are created to test the feature properly (alternator doesn't have it's own API to set it up so we need to use CQL).
+cql.execute("CREATE ROLE alternator_custom_sl WITH LOGIN = true AND HASHED PASSWORD = 'secret_pass'")
+cql.execute("CREATE SERVICE LEVEL sl_alternator")
+cql.execute("ATTACH SERVICE LEVEL sl_alternator TO alternator_custom_sl")
```

Comment:
This is so sad :-( We had such code in the past for the default role, and no longer needed it, and it's sad we needed it back. In fact, I think it's a mistake to do it in test/alternator/run, since it can actually be done *inside* the test.  See for example test_cql_rbac.py where the test itself creates the role it needs for the test.
I don't expect you to do this now, it's ok that you merge this code now because this is what we had in Enterprise - but I really want to fix it so I opened https://github.com/scylladb/scylladb/issues/22047 and plan to fix this myself after this implementation goes in.